File: python/easy/square_each_num.py
def square_each_num(num):
  if num < 10:
    return num

  result = ''
  slist = list(format(num))
  for item in slist:
    squareNum = int(item)*int(item)
    result += format(squareNum)
  return int(result)
print(square_each_num(25))  # 425
# O(n)

File: ts/easy/squareEachNum.ts
export const squareEachNum = (num: number): number => {
  if (num < 10) {
    return num;
  }
  return +num
    .toString()
    .split('')
    .map((num) => +num * +num)
    .join('');
};
console.log(squareEachNum(25)); //	425
//  O(n)
